build/org.eclipse.cdt.make.ui/plugin.xml

```diff
@@ -430,14 +430,6 @@
    </extension>
 
 
-   <extension
-         point="org.eclipse.cdt.ui.PathContainerPage">
-      <PathContainerPage
-            class="org.eclipse.cdt.make.ui.dialogs.DiscoveredPathContainerPage"
-            icon="icons/obj16/lib_obj.gif"
-            id="org.eclipse.cdt.make.core.DISCOVERED_SCANNER_INFO"
-            name="%DiscoveredScannerInfoContainer.name"/>
```

Comment:
This key needs deletion as well

Reply:
https://github.com/eclipse-cdt/cdt/pull/702